libs/ledgerjs/packages/cryptoassets/src/tokens.ts

```diff
@@ -38,6 +39,37 @@ type TokensListOptions = {
 const defaultTokenListOptions: TokensListOptions = {
   withDelisted: false,
 };
+export function createTokenHash(token: TokenCurrency): string {
+  return token
+    ? `${token.id}${token.contractAddress}${token.delisted}${token.disableCountervalue}${token.ticker}${token.countervalueTicker}${token.ledgerSignature}`
+    : "";
+}
+
+/**
+ * Only for jest purpose, clean object to be empty
+ * @param obj
+ */
+const __clearObject = (
+  obj: Record<string, TokenCurrency | TokenCurrency[]>
+): void => {
+  for (const key in obj) {
+    delete obj[key];
+  }
+};
+/**
+ * Only for jest purpose, clear all the init list
+ */
+export function __clearAllLists(): void {
+  tokensArray.length = 0;
+  tokensArrayWithDelisted.length = 0;
+  __clearObject(tokensByCryptoCurrency);
+  __clearObject(tokensByCryptoCurrencyWithDelisted);
+  __clearObject(tokensById);
+  __clearObject(tokensByTicker);
+  __clearObject(tokensByAddress);
+  __clearObject(tokensByCurrencyAddress);
+  tokenListHashes.clear();
+}
```

Comment:
I'm not sure how technically I can do this, I don't think I can with jest accessing not exported variable.

Reply:
To be more precise, I can't access to const element in `tokens.ts` because they are all in a `"private"` directly from `token.test.ts` unless I want to export all list but I won't recommended for readibility.
I'll try to find something about it

Reply:
Nevermind you're right we don't want to export all these. No big deal.